contracts/registries/LicenseNFT.sol

```diff
@@ -0,0 +1,280 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.23;
+import { Base64 } from "@openzeppelin/contracts/utils/Base64.sol";
+import { Strings } from "@openzeppelin/contracts/utils/Strings.sol";
+// solhint-disable-next-line max-line-length
+import { ERC721EnumerableUpgradeable, ERC721Upgradeable } from "@openzeppelin/contracts-upgradeable/token/ERC721/extensions/ERC721EnumerableUpgradeable.sol";
+import { IERC721Metadata } from "@openzeppelin/contracts/token/ERC721/extensions/IERC721Metadata.sol";
+import { UUPSUpgradeable } from "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+
+import { ILicenseNFT } from "../interfaces/registries/ILicenseNFT.sol";
+import { ILicensingModule } from "../interfaces/modules/licensing/ILicensingModule.sol";
+import { IDisputeModule } from "../interfaces/modules/dispute/IDisputeModule.sol";
+import { Errors } from "../lib/Errors.sol";
+import { GovernableUpgradeable } from "../governance/GovernableUpgradeable.sol";
+import { ILicenseTemplate } from "../interfaces/modules/licensing/ILicenseTemplate.sol";
+
+/// @title LicenseNFT aka LNFT
+contract LicenseNFT is ERC721EnumerableUpgradeable, GovernableUpgradeable, UUPSUpgradeable, ILicenseNFT {
```

Comment:
Why this NFT contract in the registries repo?

Reply:
Moved out from the registration directory.